shamir_mnemonic/cli.py

```diff
@@ -1,4 +1,4 @@
-import os
+import secrets
```

Comment:
this doesn't seem to be necessary

Reply:
It's actually needed below, but I didn't upload the commit correctly (you should now see it).